package.json

```diff
@@ -21,10 +21,12 @@
   },
   "homepage": "http://material-ui.com/",
   "scripts": {
-    "build": "npm run build:icon-index && npm run build:babel && npm run build:copy-files",
+    "build": "npm run build:icon-index && npm run build:babel && npm run build:copy-files && npm run build:umd && npm run build:min",
     "build:icon-index": "babel-node ./scripts/icon-index-generator.js",
     "build:babel": "babel ./src --ignore *.spec.js --out-dir ./build",
     "build:copy-files": "babel-node ./scripts/copy-files.js",
+    "build:umd": "cross-env NODE_ENV=development webpack src/index.umd.js dist/MaterialUI.js",
+    "build:min": "cross-env NODE_ENV=production webpack -p src/index.umd.js dist/MaterialUI.min.js",
```

Comment:
These two targets gave errors for me.  I changed them to:

    "build:umd": "cross-env NODE_ENV=development webpack src/index.umd.js --output-filename dist/MaterialUI.js",
    "build:min": "cross-env NODE_ENV=production webpack -p src/index.umd.js --output-filename dist/MaterialUI.min.js",

and the build ran correctly.